bitnami/discourse/templates/deployment.yaml

```
@@ -119,6 +119,12 @@ spec:
               {{- range $plugin := .Values.discourse.plugins }}
               RAILS_ENV=production bundle exec rake plugin:install repo={{ $plugin }}
               {{- end }}
+              {{- if .Values.discourse.updatePlugins }}
+              {{- if .Values.discourse.persistPlugins }}
+              RAILS_ENV=production bundle exec rake plugin:update_all
+              {{- end }}
+              RAILS_ENV=production LOAD_PLUGINS=0 bundle exec rake plugin:pull_compatible_all
+              {{- end }}
```

In this case, maybe it makes more sense to separate 'if'.What do you think?
```suggestion
              {{- if .Values.discourse.persistPlugins }}
              RAILS_ENV=production bundle exec rake plugin:update_all
              {{- end }}
              {{- if .Values.discourse.updatePlugins }}
              RAILS_ENV=production LOAD_PLUGINS=0 bundle exec rake plugin:pull_compatible_all
              {{- end }}
```

---

the and should be the same as use only un variable. 
The options are:

- Only use one variable: 
```yaml
             {{- if .Values.discourse.updatePlugins }}
                RAILS_ENV=production bundle exec rake plugin:update_all
                RAILS_ENV=production LOAD_PLUGINS=0 bundle exec rake plugin:pull_compatible_all
             {{- end }}
```
- The suggestion
```yaml
              {{- if .Values.discourse.persistPlugins }}
                RAILS_ENV=production bundle exec rake plugin:update_all
              {{- end }}
              {{- if .Values.discourse.updatePlugins }}
                RAILS_ENV=production LOAD_PLUGINS=0 bundle exec rake plugin:pull_compatible_all
              {{- end }}
```

---

The output is the same. But run of  `update_all` is useless, if plugins are not persistet, because the are alwas installed before and are at the same version.

I'm fine with your suggestion, but i would rename the variable to `compatiblePlugins`. Because it's a different use case and the plugins are always updated.

---

Ok! When you have time please update this and resolve the conflicts.